prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1715,6 +1715,7 @@ static inline void rdm_ep_poll_ibv_cq_ex(struct rxr_ep *ep, size_t cqe_to_proces
 			ep->recv_comps++;
 #endif
 			break;
+		case IBV_WC_RDMA_READ:
```

Comment:
not related to this PR, when will we remove the assert in L1688?

Reply:
I do not think we will remove the assert.

Reply:
assert(0) is something that should never happen. It helps debugging.

Reply:
I think the asset at L1688 is no longer valid. It would cause assertion failure (instead of reporting error) if a RDMA op fails.

Reply:
Sorry. I misread the code. I was talking about L1730.

L1688 is definitely wrong

Reply:
That will be handled in a separate PR